### PR TITLE
update tokenizers

### PR DESCRIPTION
**Pull request by pavel-esir**
CVS-150821

### CODE REVIEW

**Comment by pavel-esir** [comment]
rebuild

**Comment by pavel-esir** [comment]
waiting for the fix https://github.com/openvinotoolkit/openvino_tokenizers/pull/233

**Comment by pavel-esir** [comment]
Segfaults are fixed. The only remaining failure is because of extra special tokens in chat_sample, this will be fixed after switching from SentencePiece -> BPE #235

**Review by ilya-lavrenov — APPROVED** [review]

**Comment by Wovchena** [comment]
LGTM

File: image_generation/lcm_dreamshaper_v7/cpp/CMakeLists.txt [hunk 1]
```diff
@@ -15,7 +15,7 @@ set(CMAKE_BUILD_TYPE "Release" CACHE STRING "CMake build type")
 
 # dependencies
 
-find_package(OpenVINO REQUIRED COMPONENTS Runtime)
+find_package(OpenVINO REQUIRED COMPONENTS Runtime Threading)
```
Comment (ilya-lavrenov) [inline image_generation/lcm_dreamshaper_v7/cpp/CMakeLists.txt:18]: FYI: will be reverted once https://github.com/openvinotoolkit/openvino/pull/26269 is merged and used within GenAI

**Review by apaniukov — APPROVED** [review]

**Pull request merged**